File: works/1427.cpp
#include <iostream>
#include <string>
#include <vector>
#include <algorithm>
using namespace std;

int main()
{
    string num; cin >> num;
    vector<int> v(num.size());
    for(int i = 0; i < num.size(); i++) v[i] = num[i]-'0';
    sort(v.begin(), v.end(), greater<>());
    for(auto x : v) cout << x;
}

File: works/1427.kt
import java.util.*


fun main()
{
    val sc = Scanner(System.`in`)
    val num = sc.next()
    var v = Array<Int>(num.length) {0}
    for(i in 0 until num.length) v[i] = num[i]-'0'
    Arrays.sort(v, Collections.reverseOrder())
    for(x in v) print(x)
}